Fixup DJANGO_SETTINGS_SKIP_LOCAL in tests

xrmx opened the pull request:
The import path for the local_settings should be relative to the
test module and not absolute otherwise the import will fail.

humitos commented:
What was the error that you were receiving without this patch?

How I can reproduce it?

xrmx commented:
Just add some options in local_settings.py, run the tests and you see they are ignored. In my case i've added a DATABASES entry, removed DJANGO_SETTINGS_SKIP_LOCAL=False from tox.ini and run tox.

xrmx commented:
And also `settings/dev.py` use the dotted import :)

agjohnson commented:
This looks fine, as it mimics `dev.py`. It's better to rely on a custom settings file for your tests however, not `local_settings`.
